Sources/TokamakCore/Fiber/ViewArguments.swift

```diff
@@ -33,7 +37,11 @@ public struct ViewOutputs {
   /// This is stored as a reference to avoid copying the environment when unnecessary.
   let environment: EnvironmentBox
 
-  let preferences: _PreferenceStore
+  let preferenceStore: _PreferenceStore?
+
+  let transformPreferences: ((_PreferenceStore) -> ())?
+
+  let preferenceAction: ((_PreferenceStore) -> ())?
```

Comment:
these two definitely need a doc comment to clarify the difference between them, as they have the same time, which doesn't make it easier to infer their purpose